core/src/targeting/build-page-targeting.ts

```diff
@@ -136,13 +141,18 @@ const buildPageTargeting = ({
 		youtube,
 	});
 
+	const otherTargeting = {
```

Comment:
Was wondering if rather than introducing `otherTargeting` we could derive `allkw` in `contentTargeting` by shifting the order and passing it `k`?

Reply:
yeah, sounds good to me! Wasn't sure where to put it tbh